shared/js/option_menu.js

```diff
@@ -131,6 +131,15 @@ var OptionMenu = function(options) {
       return;
     }
     menu.appendChild(button);
+
+    // Wrap the last button in a container in order to style it differently.
+    if (index === items.length - 1) {
+      var lastButtonWrapper = document.createElement('span');
+      lastButtonWrapper.className = 'button-container';
```

Comment:
or something better than just "button-container"; "last-button-container" would work for me too.